src/module-communication.md

```diff
@@ -0,0 +1,47 @@
+# Module Communication
+
+This document describes communication between the high level modules of the poet node.
+
+## Writing Claims
+
+- initial
```

Comment:
Maybe add > User posts a new work to node using `POST /works`?

Reply:
Added to the line below

```
-initial
 - **API** User posts a new work to node using `POST /works`, publishes `newClaim`
```